src/materialized/src/version_check.rs

```diff
@@ -7,30 +7,33 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use std::time::Duration;
+use std::collections::HashSet;
+use std::time::{Duration, Instant};
 
 use anyhow::bail;
 use log::{debug, log, Level};
 use semver::{Identifier, Version};
 use serde::{Deserialize, Serialize};
+use tokio::time::{sleep as tokio_sleep, Duration as TokioDuration};
```

Comment:
There's actually no need to rename these imports! `std::time::Duration` and `tokio::time::Duration` are the same type. Idiomatic usage of Tokio's time module involves importing like so

```rust
use tokio::time::{self, Duration, Instant};
```

and then you can use

```
time::sleep(Duration::from_secs(1))
```

and such throughout.